notification/src/test/java/com/kyriba/conference/notification/api/SmsNotificationControllerTest.java

```diff
@@ -18,12 +23,27 @@
 import static junit.framework.TestCase.assertEquals;
 import static junit.framework.TestCase.assertNotNull;
 import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
+import static org.springframework.restdocs.restassured3.RestAssuredRestDocumentation.document;
+import static org.springframework.restdocs.restassured3.RestAssuredRestDocumentation.documentationConfiguration;
 
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
 public class SmsNotificationControllerTest
 {
+  @Rule
+  public final JUnitRestDocumentation restDocumentation = new JUnitRestDocumentation();
+
+  private RequestSpecification documentationSpec;
+
+
+  @Before
```

Comment:
You could move duplicated pieces in common base class for all tests of such type.